client/me/memberships/main.jsx

```diff
@@ -84,7 +84,13 @@ const MembershipsHistory = ( { translate, subscriptions, moment } ) => {
 	if ( subscriptions && subscriptions.length ) {
 		content = (
 			<>
-				<SectionHeader label={ translate( 'Active Recurring Payments plans' ) } />
+				<SectionHeader
+					label={
+						isEnabled( 'earn/rename-payment-blocks' )
+							? translate( 'Active payments plans' )
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.